vitest.browser.config.mts

```diff
@@ -20,17 +20,25 @@ const config: UserConfig = {
         "timers/promises",
         "util",
         "stream",
+        "path",
+        "fs",
+        "os",
       ],
       overrides: {
         fs: "memfs",
       },
     }),
   ],
   optimizeDeps: {
-    exclude: ["fsevents", "path-scurry"],
+    exclude: ["fsevents", "path-scurry", "@vitest/coverage-istanbul"],
     include: ["events", "timers/promises"],
+    entries: ["**/*.test.ts"],
```

Comment:
Most investigation seems to point to flaky ESM bundling. Haven't had any failed browser runs in 10 attempts after this change.